dev_guide/secrets.adoc

```diff
@@ -22,8 +22,14 @@ private source repository credentials, and so on. Secrets decouple sensitive
 content from the pods. You can mount secrets into containers using a volume
 plug-in or the system can use secrets to perform actions on behalf of a pod.
 
+You can specify `type: Opaque` to configure An _opaque_ secret, which allows for 
+unstructured `key:value` pairs that can contain arbitrary values. If you do not
+specify `opaque`, the values in a secret have constrained contents. An opaque secret represents the internal version of this object. An opaque secret 
```

Comment:
> An opaque secret represents the internal version of this object

I don't understand what that means.

There are specific types of secrets we recognize and make use of (`kubernetes.io/service-account-token`, `kubernetes.io/dockercfg`, etc).

Someone creating secrets for other purposes can either set type to a value of their choosing (like `acme.com/my-secret-type`) or leave it unset, and it defaults to `Opaque`.

Reply:
> An opaque secret represents the internal version of this object

This comes from the Kubernetes docs. I removed it as I don't know what it means,either!